mautic/mautic
# Add referer to Visit a page campaign decision

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n
| New feature? | y
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/136
| Related developer documentation PR URL | n
| Issues addressed (#s or URLs) | n
| BC breaks? | n
| Deprecations? | n

#### Description:

Small but very useful improvements. Now we can build campaign with referer  page hit. 
We can make marketing automation based on referer (google, facebook...)

![visitpage](https://cloud.githubusercontent.com/assets/462477/20136286/3f0e92cc-a674-11e6-9431-b650296630ea.PNG)

#### Steps to test this PR:
1. Create campaign with Visit page decision, fill a referer field and add test action after this decision (add tag for example)
2. After visit page check. Works If your action was processed
